### DISCUSSION TITLE
Home Page Dashboards Disappeared for User

### DISCUSSION THREAD

**Discussion by alb3rtod** [chartbrew/chartbrew#254] [Q&A] [answered]
Hi @razvanilin 

I have a user who was initially able to see the dashboards on the home page. However, now the home page appears empty, as shown in the attached image. What could be causing this issue?

<img width="580" alt="image" src="https://github.com/user-attachments/assets/17ff8356-f229-43bf-8e6f-b4a3396cdf82">

**Comment by razvanilin** [comment]
Hey @alb3rtod, I would need more information to be able to find out what's happening. Some questions:

1. Are there any errors in the browser's devtools?
2. Have you deleted the team the user is part of?
3. Have you interfered with the database manually outside Chartbrew UI?
4. Can you access any of the dashboards through a direct link rather than on the home page?

**Reply by alb3rtod** [reply, marked as answer]
Hi @razvanilin 

1. Are there any errors in the browser's devtools? No, there aren't any in my opinion.
2. Have you deleted the team the user is part of? No, I haven't
3. Have you interfered with the database manually outside Chartbrew UI? No, I haven't
4. Can you access any of the dashboards through a direct link rather than on the home page?
Yes! that actually works. Thank you! After trying to access the dashboards through a direct link, it seems like the issue is fixed. What could have happened?

**Reply by razvanilin** [reply]
hmm that's quite strange, but should give me some hints to find the source of the issue. Let me know if you notice this happening again
